src/Omega_h_array.hpp

```diff
@@ -162,7 +162,21 @@ class HostWrite {
   using value_type = T;
   HostWrite() = default;
   HostWrite(LO size_in, std::string const& name = "");
+  
+  /**
+ * \brief Constructs a HostWrite object with specified size, offset, and stride.
+ *
+ * \param size_in The number of elements in the array.
```

Comment:
Given that this is a mesh adaptation library, I think we should avoid use of the term 'element' if it is not specifically being used to reference a mesh element.  Replacing 'element(s)' with 'entry' in could work; I'm open to alternatives/suggestions.

Reply:
"Entry" sounds good to me. I have amended the commit.